exercises/all-your-base/all-your-base.sml

```diff
@@ -0,0 +1,2 @@
+fun rebase (fromBase: int) (xs: int list) (toBase: int): int list option =
+  raise Fail "'rebase' has not been implemented"
```

Comment:
Hmm, `bin/generate` uses `"{function} is not implemented"`. In order to keep consistency, I would not change it. But if you think `"'{function}' has not been implemented` is better, please mention your reasons therefore we can update the the script.